source-monday/tests/test_snapshots.py

```diff
@@ -17,7 +17,7 @@ def test_capture(request, snapshot):
             "--sessions",
             "1",
             "--delay",
-            "60s",
+            "250s",
```

Comment:
Updated since API rate limits cause delays and failures in tests sometimes. This is pretty common on a free account.